courses/free-courses-en.md

```diff
@@ -242,6 +242,7 @@
 
 ### <a id="csharp"></a>C\#
 
+* [Building a microservice architecture with ASP.NET Core](https://www.youtube.com/watch?v=SR53SKIUYPA&list=PL03Lrmd9CiGfgBmqFWeRxaUSQ_cSF_rCF&index=30) - Gill Cleeren (Youtube)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
As requested by Eric...

```suggestion
* [Building a microservice architecture with ASP.NET Core - Gill Cleeren - NDC London 2022](https://www.youtube.com/watch?v=SR53SKIUYPA) - Gill Cleeren
```